test/helpers/generic_dao.js

```diff
@@ -24,8 +24,13 @@
 
 
 global.genericDAOTestBattery = function(daoFactory) {
+  var oldContext;
+
   describe('generic DAO tests', function() {
     beforeEach(function() {
+      oldContext = foam.__context__;
```

Comment:
Nested suites were a problem at one point, but I'm not seeing errors related to that anymore. Removing